### ISSUE TITLE
Error after used "date phrase"

### ISSUE THREAD

**Issue by hartenthaler** [fisharebest/webtrees#4944]
In the [Forum](https://www.webtrees.net/index.php/forum/help-for-release-2-1-x/38079-webtrees-2-1-16-signalement-dune-fausse-anomalie#103512) a user showed that he is using
```
0 @F9098@ FAM
1 HUSB @I26161@
1 WIFE @I26160@
1 MARR
2 DATE (../../....)
```
The DATE is legal GEDCOM because GEDCOM 5.5.1 says
```
DATE_VALUE:= {Size=1:35}
[
<DATE> |
<DATE_PERIOD> |
<DATE_RANGE>|
<DATE_APPROXIMATED> |
INT <DATE> (<DATE_PHRASE>) |
(<DATE_PHRASE>)
]
```
So it is not ok that webtrees shows the marriage event with an error flag and the comment 
[( after death)](https://www.brainans-notre-histoire.fr/genealogie/tree/Brainans/individual/I16771/Gregoire-CHERIN#)

**Comment by fisharebest** [comment]
See my comment in the forum.  I cannot reproduce this.  Can you provide a GEDCOM that does?

**Comment by hartenthaler** [comment]
[wtcce(I31071).ged.txt](https://github.com/fisharebest/webtrees/files/14016402/wtcce.I31071.ged.txt)
Look at the person "Stief GC5 Irgendwas".

**Comment by fisharebest** [comment]
```
0 HEAD
1 GEDC
2 VERS 5.5.1
2 FORM LINEAGE-LINKED
1 CHAR UTF-8
1 FILE dates.ged
0 @I26160@ INDI
1 NAME aaa /AAA/
2 GIVN aaa
2 SURN AAA
1 SEX F
1 BIRT
2 DATE 28 MAY 1928
1 DEAT
2 DATE 24 JAN 2021
1 FAMS @F9098@
1 FAMC @F3352@
0 @I26161@ INDI
1 NAME bbb /BBB/
2 GIVN bbb
2 SURN BBB
1 SEX M
1 BIRT
2 DATE 29 AUG 1921
1 FAMS @F9098@
0 @F3352@ FAM
1 CHIL @I26160@
1 MARR
2 DATE 13 OCT 1926
0 @F9098@ FAM
1 HUSB @I26161@
1 WIFE @I26160@
1 MARR
2 DATE (../../....)
0 TRLR
```

**Comment by arbor95** [comment]
see pull request #4946

**Comment by fisharebest** [comment]
Fixed by #4946
